File: service/src/main/java/hu/bsstudio/robonaut/race/skill/timer/SkillTimerService.java
package hu.bsstudio.robonaut.race.skill.timer;

import hu.bsstudio.robonaut.race.skill.timer.model.SkillTimer;
import reactor.core.publisher.Mono;

public interface SkillTimerService {
    Mono<SkillTimer> startTimer(final SkillTimer skillTimer);

    Mono<SkillTimer> stopTimer(final SkillTimer skillTimer);
}

File: service/src/main/kotlin/hu/bsstudio/robonaut/race/skill/timer/SkillTimerService.kt
package hu.bsstudio.robonaut.race.skill.timer

import hu.bsstudio.robonaut.race.skill.timer.model.SkillTimer
import reactor.core.publisher.Mono

interface SkillTimerService {
    fun updateTimer(skillTimer: SkillTimer): Mono<SkillTimer>
}
